diff --git a/dist/constants.cjs b/dist/constants.cjs
index 186045d8d2085df4db0afca8cb74bea01c427edc..70eb00e6ff6feebb5bb39d796f4f2691006a02f9 100644
--- a/dist/constants.cjs
+++ b/dist/constants.cjs
@@ -351,6 +351,7 @@ function isSingleLetterKey(key) {
 	return /^\p{Letter}$/u.test(key);
 }
 function normalizeKeyName(key) {
+	if (typeof key !== "string") return "";
 	if (key in KEY_ALIASES) return KEY_ALIASES[key];
 	if (isSingleLetterKey(key)) return key.toUpperCase();
 	const upperKey = key.toUpperCase();
diff --git a/dist/constants.js b/dist/constants.js
index 1bde9bd6b8208c05c741d7173cee075fb8d7b1b0..1cc1fd11eaa4ab8c98c0a492223ed20244a5c486 100644
--- a/dist/constants.js
+++ b/dist/constants.js
@@ -350,6 +350,7 @@ function isSingleLetterKey(key) {
 	return /^\p{Letter}$/u.test(key);
 }
 function normalizeKeyName(key) {
+	if (typeof key !== "string") return "";
 	if (key in KEY_ALIASES) return KEY_ALIASES[key];
 	if (isSingleLetterKey(key)) return key.toUpperCase();
 	const upperKey = key.toUpperCase();
